docker/Makefile

```
@@ -95,6 +96,21 @@ else
 	@echo "==> Postgres container already stopped before."
 endif
 
+build: ## Build the container image
```

checkmake (0.3.2)

[warning] Target body for "build" exceeds allowed length of 5 lines (13).

(maxbodylength)